cmd/influxd/run.go

```diff
@@ -23,7 +23,7 @@ import (
 	"github.com/influxdb/influxdb/udp"
 )
 
-func Run(config *Config, join, version string) (*messaging.Broker, *influxdb.Server) {
+func Run(config *Config, join, version string) (*messaging.Broker, *influxdb.Server, *raft.Log) {
```

Comment:
I feel like the return values are getting a little bit out of control. Perhaps we need a more generic `node` type which returned by `Run()`. This `node` type would have a `Close()` method, which would take of closing whatever was opened. Sometimes `node` would have a broker, sometimes a log, sometimes a server, sometimes all, etc.